torch/nn/modules/pooling.py

```diff
@@ -1094,6 +1094,20 @@ class AdaptiveMaxPool3d(_AdaptiveMaxPoolNd):
         - Output: :math:`(N, C, D_{out}, H_{out}, W_{out})` or :math:`(C, D_{out}, H_{out}, W_{out})`,
           where :math:`(D_{out}, H_{out}, W_{out})=\text{output\_size}`.
 
+    Behavior:
```

Comment:
For future reference, yuou can check the rendering of this in the docs preview linked in the `pytorch-bot` comment on this PR

> 📄 Preview [Python docs built from this PR](https://docs-preview.pytorch.org/pytorch/pytorch/109432/index.html)

1) Seems like you intended this change to be for `AdaptiveMaxPool2d` but added it to `AdaptiveMaxPool3d`

2) Could you include the behavior in the paragraph at the top that describes the function instead of a separate behavior section